README.md

```diff
@@ -15,17 +15,48 @@ This will ensure that the `PYTHONPATH` will be set correctly.
 
 ## Launch experiments
 
-Open and modify `exp_configs.py` and `launch_command.py` to your needs. They are
-located in `agentlab/experiments/`.
+We provide default settings to run our agents on a few benchmarks. Those can be ran from `launch_exp.py`,
+with the following flags:
+
+```bash
+  -h, --help            show this help message and exit
+  --exp_root EXP_ROOT   folder where experiments will be saved
+  --n_jobs N_JOBS       number of parallel jobs
+  --exp_group_name EXP_GROUP_NAME
+                        Name of the experiment group to launch as defined in exp_configs.py
+  --benchmark {miniwob,workarena.l1,workarena.l2,workarena.l3}
+                        Benchmark to launch
+  --model_name {gpt-3.5,gpt-4o,gpt-4o-vision,cheat,custom}
+                        Model to launch
+  --relaunch_mode {None,incomplete_only,all_errors,server_errors}
+                        Find all incomplete experiments and relaunch them.
+```
+
+`exp_group_name` is the name of the experiment group to launch as defined in `exp_configs.py`. This will override the `benchmark` and `model_name` flags.
+
+As an example, to launch our agent with GPT-4o on the miniwob benchmark, with maximum jobs, run:
+
+```bash
+    python src/agentlab/experiments/launch_exp.py  \
+        --benchmark=miniwob \
+        --model_name=gpt-4o \
+        --n_jobs=-1
+```
+
+The `cheat` model uses regular expressions to solve miniwob click tasks. It is useful for debugging.
+
+In `exp_configs.py`, you can modify `FLAGS_CUSTOM` and `AGENT_CUSTOM` to test out your own flags and models, and then launch them with `--model_name=custom`.
+
+### Custom experiments
+
+Alternatively, you can customize your experiments by modifying `exp_configs.py` and `launch_command.py`. They are located in `agentlab/experiments/`.
 
 Then launch the experiment with
 
 ```bash
-    python launch_command.py
+    python src/agentlab/experiments/launch_command.py
 ```
 
-Avoid pushing these changes to the repo unless they are structural changes.
-If you prefer launching with command line, see section [Launch experiments with command line](#launch-experiments-with-command-line).
 
 ### Debugging jobs
 
```

Comment:
I know it's not what you modified, but could you remove "use_threads...=True", this leads to weird behavior. They should set n_jobs=1 for debugging.